### ISSUE TITLE
Unable to Connect

### ISSUE THREAD

**Issue by prakharsingh** [MrAsterisco/homebridge-hisense-tv#13]
Homebridge Logs:

`[18/06/2021, 02:34:43] [HiSenseTV] An error occurred while fetching inputs: TypeError: Cannot read property 'join' of null`

Terminal Logs: 

```
python3.8 /usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/hisensetv.py 192.168.1.239 --ifname wlan0 --get sources
[2021-06-18 02:31:25,851] [INFO    ] Unverified SSL context created.
[
    {
        "displayname": "TV",
        "hotel_mode": "0",
        "is_lock": "false",
        "is_signal": "1",
        "sourceid": "1",
        "sourcename": "TV"
    },
    {
        "displayname": "Cable",
        "hotel_mode": "0",
        "is_lock": "false",
        "is_signal": "1",
        "sourceid": "2",
        "sourcename": "HDMI 1"
    },
    {
        "displayname": "Game console",
        "hotel_mode": "0",
        "is_lock": "false",
        "is_signal": "0",
        "sourceid": "3",
        "sourcename": "HDMI 2"
    },
    {
        "displayname": "AV",
        "hotel_mode": "0",
        "is_lock": "false",
        "is_signal": "0",
        "sourceid": "4",
        "sourcename": "AV"
    }
]
pi@raspberrypi:~ $ python3.8 /usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/hisensetv.py 192.168.1.239 --ifname wlan0 --get state
[2021-06-18 02:31:45,621] [INFO    ] Unverified SSL context created.
Traceback (most recent call last):
  File "/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/hisensetv/__init__.py", line 208, in _wait_for_response
    return self._queue[topic].get(block=True, timeout=self.timeout)
  File "/usr/local/lib/python3.8/queue.py", line 178, in get
    raise Empty
_queue.Empty

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/hisensetv.py", line 6, in <module>
    __main__.main()
  File "/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/hisensetv/__main__.py", line 141, in main
    output = func()
  File "/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/hisensetv/__init__.py", line 56, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/hisensetv/__init__.py", line 661, in get_state
    return self._wait_for_response(topic=self._BROADCAST_TOPIC)
  File "/usr/local/lib/node_modules/homebridge-hisense-tv-remotenow/bin/hisensetv/__init__.py", line 210, in _wait_for_response
    raise HisenseTvTimeoutError(
hisensetv.HisenseTvTimeoutError: failed to recieve a response in 10.000s
```

**Comment by MrAsterisco** [comment]
Hey @prakharsingh ! Thanks for reporting this.

Have you already tried messing around with the SSL options? What I see in the logs is that you are not using a SSL context, but some TVs require it.

**Comment by prakharsingh** [comment]
Hey @MrAsterisco Thanks for the suggestion. Adding SSL certs helped (though they didn't worked when I tried earlier so I thought my TV didn't needed any certs)
I'd like to mention for anyone trying this plugin that python3.8 installation should be checked on the homebridge user as well. I ran into this problem where the pi user had installed ptyhon3.8 in /usr/local/lib. Post that I had to install paho-mqtt on both the users and finally everything worked as expected.
